app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminTaxRateGridOpenPageActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminTaxRateGridOpenPageActionGroup">
```

Comment:
I would suggest to rename it, so we have a verb-action after the area name. Most likely, developer will look rather for AdminOpen... than for AdminTax (since the entire module is about tax).

```suggestion
    <actionGroup name="AdminOpenTaxRateGridPageActionGroup">
```

What do you think?

Reply:
Hi @rogyar 
Yes, I can do it . If I don't get another request from @lbajsarowicz  to change name of action group  to `AdminTaxRateGridOpenPageActionGroup` again 😄  
Like it was in https://github.com/magento/magento2/pull/28925 
Should I do it ?

Reply:
You are right. That's a very debatable topic from different perspectives. Let's leave it as it is for now.